ocs/pkg/service/v0/users.go

```diff
@@ -358,6 +368,126 @@ func (o Ocs) DeleteUser(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
```

Comment:
Shouldn't be there a config option for `STORAGE_USERS_DRIVER` too?

Reply:
yes, still not done with this, just asserting that this way of deprovisioning is viable 😸

Reply:
in hindsight doing this from an env var is reckless haha